src/presenter/event-presenter.js

```diff
@@ -0,0 +1,24 @@
+import { render } from '../render.js';
+import EventEditPointView from '../view/event-edit-point-view.js';
+import EventListView from '../view/event-list-view.js';
+import EventSortView from '../view/event-sort-view.js';
+import EventView from '../view/event-view.js';
+
+export default class EventPresenter {
+  constructor({ eventContainer }) {
+    this.eventContainer = eventContainer;
+  }
```

Comment:
Нет смысла добавлять лишние слова, усложнять читаемость без особой необходимости.
У нас всего один контейнер в компоненте, поэтому и без main понятно

```suggestion
  constructor({ container }) {
    this.container = container;
  }
```